src/liblibc/lib.rs

```diff
@@ -227,7 +227,7 @@ pub mod types {
                 pub type rlim_t = u64;
 
                 #[repr(C)]
-                #[derive(Copy)] pub struct glob_t {
+                #[derive(Copy, Clone)] pub struct glob_t {
```

Comment:
@alexcrichton why? (that is, why is this any more unfortunate than any other fallout?)

seems like this will be a lot less painful for everyone if we made `#[derive(Copy)]` imply `#[derive(Clone)]`...